test/extensions/filters/http/jwt_authn/authenticator_test.cc

```diff
@@ -185,6 +187,59 @@ TEST_F(AuthenticatorTest, TestMissedJWT) {
   expectVerifyStatus(Status::JwtMissed, headers);
 }
 
+// Test multiple tokens; one of them is bad, verification is ok.
+TEST_F(AuthenticatorTest, TestMultipleJWTOneBad) {
+  EXPECT_CALL(*raw_fetcher_, fetch(_, _, _)).Times(1);
+
+  // headers with multiple tokens: one good, one bad
+  auto headers = Http::TestRequestHeaderMapImpl{
```

Comment:
```suggestion
  Http::TestRequestHeaderMapImpl headers{
```

style nit, same for below

Reply:
Done